demos/build_demos.sh

```diff
@@ -82,6 +88,7 @@ fi
 mkdir -p "$build_dir"
 
 (cd "$build_dir" && cmake -DCMAKE_BUILD_TYPE=Release "${extra_cmake_opts[@]}" "$DEMOS_PATH")
-cmake --build "$build_dir" -- "$NUM_THREADS"
+c="cmake --build $build_dir ${extra_cmake_build_opts[@]} -- $NUM_THREADS"
+eval "$c"
```

Comment:
Why not
```suggestion
cmake --build $build_dir ${extra_cmake_build_opts[@]} -- $NUM_THREADS
```
?

Reply:
Well, I tried next variant:  
`cmake --build "$build_dir" "${extra_cmake_build_opts[@]}" -- "$NUM_THREADS`"
and it doesn't work, probably because "${extra_cmake_build_opts[@]}" contains spaces inside and bash can't resolve it without eval command.
But without quotes all is ok, I'll change it.

Reply:
May be the quotes were useful. I didn't notice you removed them :)